tests/test_node_pool.py

```diff
@@ -192,6 +192,11 @@ def test_dead_node_backoff_calculation():
     assert pool._dead_consecutive_failures == {node.config: 4}
     assert pool._dead_nodes.queue == [(3.5, node)]
 
+    assert pool.get() is node
+    pool._dead_consecutive_failures = {node.config: 13292}
+    pool.mark_dead(node, _now=0)
+    assert pool._dead_nodes.queue == [(3.5, node)]
```

Comment:
Super small nit to keep consistency with the rest of this test. This does not need another round of reviews.

```suggestion
    assert pool.get() is node
    pool.mark_dead(node, _now=0)
    
    pool._dead_consecutive_failures = {node.config: 13292}
    assert pool._dead_nodes.queue == [(3.5, node)]
```